apps/examples/wifi_manager/wm_test/wm_test_dtls_client.c

```diff
@@ -0,0 +1,292 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License\n");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#include <sys/types.h>
+#include <stdio.h>
+#include <string.h>
+#include "wm_test_log.h"
+#include "mbedtls/config.h"
+#include "mbedtls/net.h"
+#include "mbedtls/debug.h"
+#include "mbedtls/ssl.h"
+#include "mbedtls/entropy.h"
+#include "mbedtls/ctr_drbg.h"
+#include "mbedtls/error.h"
+#include "mbedtls/certs.h"
+#include "mbedtls/timing.h"
+
+#define SERVER_PORT "4433"
+#define READ_TIMEOUT_MS 1000
+#define MAX_RETRY 5
+#define DEBUG_LEVEL 0
+#define WT_BUF_SIZE 1024
+#define TAG "[WTDC]"
```

Comment:
Can we write all stress5 related tags in same format
WT_STRESS_5
WT_STRESS_5_CLIENT
WT_STRESS_5_DTLS_C
WT_STRESS_5_DTLS_S

Reply:
Changes done as suggested